### PR TITLE
[FLOC-2712] Get rid of usage of clusterhq/flaskenv in acceptance tests.

### PR DESCRIPTION
**Pull request by itamarst**

### CODE REVIEW

File: flocker/acceptance/test_api.py [hunk 1]
```diff
@@ -89,46 +90,25 @@ def test_create_container_with_environment(self, cluster):
         Create a container including environment variables on a single-node
```
Comment (adamtheturtle) [inline flocker/acceptance/test_api.py]: This docstring has made the review of this issue harder - what I should be looking at mostly is whether the new test fits the description. Instead I am grokking the old test and looking at the new test to make sure that it is missing nothing I guess is important. I'm working on the assumption that it should be more like:

"If environment variables are specified when creating a container, those variables are available in the container's environment".

**Comment by adamtheturtle** [comment]
I think that https://github.com/ClusterHQ/docker-images/tree/master/flaskenv is now not used. Add a note to the README that it is a legacy image.

File: flocker/acceptance/envhttp.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+"""
+HTTP server that returns its environment variables as JSON.
+"""
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+from json import dumps
+from os import environ
+
+
+class Handler(BaseHTTPRequestHandler):
```
Comment (adamtheturtle) [inline flocker/acceptance/envhttp.py]: Docstrings

**Comment by adamtheturtle** [comment]
Thanks @itamarst , please address the very minor comments and merge when Buildbot is green.

**Comment by itamarst** [comment]
In theory this shoudl fix things, in practice have some local test failures. We'll see if buildbot is happy.

**Comment by itamarst** [comment]
@exarkun -^

**Pull request merged**